src/app/Http/Controllers/Operations/ListOperation.php

```diff
@@ -72,7 +72,7 @@ public function search()
         $this->crud->hasAccessOrFail('list');
 
         // how many total entries there is applying the controller/model constrains, before any filtering or search
-        $this->crud->setOperationSetting('totalEntryCount', $this->crud->getTotalEntryCount());
```

Comment:
Hmmm I still don't think we should pass the request as a parameter here... We can do:

```php
$totalEntryCount = request()->has('totalEntryCount') ? request('totalEntryCount') : $this->crud->getTotalEntryCount();
$this->crud->setOperationSetting('totalEntryCount', $totalEntryCount);
```

Why bloat up `getTotalEntryCount()` with working with the request? I see that method as a CrudPanel method, so it's got nothing to do with the request, it should just answers what's inside CrudPanel, as a setting... otherwise calculate it, store it as the setting, then return it... right?